include/RAJA/util/basic_mempool.hpp

```diff
@@ -403,7 +403,7 @@ class MemPool {
 
   void free(const void* cptr)
   {
-#if defined(RAJA_ENABLE_OPENMP) && defined(_OPENMP)
+#if defined(RAJA_ENABLE_OPENMP)
```

Comment:
Maybe @MrBurmark has some insight here, but I believe there was a special reason why defined(_OPENMP) was added to this

Reply:
@willkill07 @trws had a good suggestion to use RAJA_ENABLE_OPENMP in the code for consistency with other PMs and make it an alias to _OPENMP. @davidbeckingsale created a JIRA issue for this.

Reply:
I believe the idea was to allow people to configure with OpenMP but only use non-OpenMP parts of RAJA and compile without OpenMP. @trws 's idea sounds good to me. I assume it will be aliased only when configuring with OpenMP, so we aren't potentially using OpenMP all the time.